website/themes/diflabs/assets/css/main.css

```diff
@@ -201,11 +200,10 @@ footer {
 a {
     color: #66FF66; /* Change link color to light green */
     text-decoration: none;
-    border-bottom: 2px dashed #66FF66; /* Add dashed underline */
 }
 
 a:visited {
-    color: #ADD8E6; /* Change visited link color to yellow */
+    color: #00BF00; /* Change visited link color to yellow */
```

Comment:
This changes link colour to green for visited links, i.e., same as non-visited links. Intentional?

Reply:
It's a slightly different tone. The ADD8E6 that was previously there is not thematically right. 00BF00 a different color than 00ff00